### PR TITLE
7.0 add web export xml

### PR DESCRIPTION
**Pull request by ntodorova**
Add new new module web_export_xml, which extends the export core functionality and allows to export records in xml format.

### CODE REVIEW

**Comment by ntodorova** [comment]
The module is still a demo version. Some collaboration, to clean the bugs and improve it, will be appreciated. : )

**Comment by OSguard** [comment]
rebase this MR to fix runbot

**Comment by OSguard** [comment]
@max3903 this module can help you to export data for module_prototyper

File: web_export_xml/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "XML Data Module Export",
+    "version": '0.1',
```
Comment (OSguard) [inline web_export_xml/__openerp__.py]: Version: 7.0.1.0
please change

**Pull request closed without merging**